module.xml

```diff
@@ -2,8 +2,7 @@
 <Export generator="Cache" version="25">
 <Document name="ZPM.ZPM"><Module>
   <Name>ZPM</Name>
-  <GlobalScope>true</GlobalScope>
-  <Version>0.9.0+snapshot</Version>
+  <Version>0.9.0-SNAPSHOT</Version>
```

Comment:
Questions: 
What does `snapshot` typically mean in package definitions. Does it mean "development in progress"?
What is the reason for changing the casing and switching from `+build` to `-prerelease`?

Reply:
Yes, it means "this is not a released version" - although our usage and rules around this are a little different from standard semantic versioning.

The standard in Maven (for Java, one of IPM's inspirations) is to use -SNAPSHOT for the current in-development version. See e.g. https://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it

You don't want to just call your current in-development version "0.9.0" because if an artifact for it ends up somewhere, it could be mistaken for the real 0.9.0. In the initial package manager implementation I chose to break with the semver standard and distinguish unreleased software via build metadata (+snapshot) rather than a prerelease (-SNAPSHOT). I thought there was some good reason for this at the time, but I don't remember it and now think it's silly to use +snapshot. The automation in GitHub actions for IPM assumes -SNAPSHOT so I've changed it back.

cc @isc-kiyer @isc-bzhang